components/elements/AccountDropdown.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import styled from 'styled-components';
+import useUser from '../auth/User';
+
+const AccountDropdown = ({ children }) => {
+  const user = useUser();
+
+  let userInitial;
+  if (user && user.name) {
+    userInitial = user.name[0].toUpperCase();
+  }
+  return (
+    <>
+      <Circle>
+        <div className="initial">{userInitial}</div>
+        <div className="menu">{children}</div>
+      </Circle>
+    </>
+  );
+};
+
+const Circle = styled.div`
```

Comment:
putting everything inside of this style components takes away the use of styled component. 
It's in the name style **COMPONENT**; which are small specific and reusable blocks

Reply:
Also see if you can give it more context instead of `div`